executor/executor.go

```diff
@@ -0,0 +1,64 @@
+package executor
+
+import (
+	"bufio"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// ParseCommand takes the command string and split it with the " " separator.
+// The first element will be the basecommand and the others will be the args.
+func ParseCommand(command string) (string, []string) {
+	commandSlice := strings.Split(command, " ")
+	name := commandSlice[0]
+	args := commandSlice[1:]
+	return name, args
+}
+
+// GetWriteFile accepts a filename and tries to open it for Writing.
+// It returns a pointer to the file, and an optional error (if it fails to get the required file).
+func GetWriteFile(fileName string) (*os.File, error) {
+	file, err := os.OpenFile(fileName, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0600)
+	if err != nil {
+		err = errors.Wrapf(err, "Cannot open outputFile %v for write", fileName)
```

Comment:
`return nil, errors.Wrapf(err, "Cannot open outputFile %v for write", fileName)`